drivers/uktty/pl011/exportsyms.uk

```diff
@@ -1,3 +1,4 @@
+pl011_early_init
```

Comment:
Commit message of
> drivers/uktty/pl011: Add early init

Add early device init. When paging is enabled, this adds an mrd for
the pl011 MMIO region to bootinfo so that the region is **mapped after paged memory init**.

Reply:
Currently the commit message states:
> region is not unmapped during paged memory init.

That is actually accurate. MMIO regions of early devices are already mapped, and the purpose of adding an mrd to bootinfo is so that the device does not get unmapped during paged memory init.